src/category_theory/currying.lean

```diff
@@ -89,6 +89,7 @@ def curry : ((C × D) ⥤ E) ⥤ (C ⥤ (D ⥤ E)) :=
 /--
 The equivalence of functor categories given by currying/uncurrying.
 -/
+@[simps {rhs_md := semireducible}]
```

Comment:
Perhaps configuration option is obscure enough that it warrants a comment here why it is needed?

Reply:
```suggestion
@[simps {rhs_md := semireducible}] -- create projection simp lemmas even though this isn't a `{ .. }`.
```